custom_components/babybuddy/services.yaml

```diff
@@ -21,108 +21,53 @@ services_children_add:
       selector:
         text:
 
-services_delete:
-  name: Delete Entry
-  description: Deletes a Baby Buddy entry
+start_timer:
+  name: Start Timer
+  description: Start a new timer
+  target:
+    entity:
+      integration: babybuddy
+      domain: switch
   fields:
-    endpoint:
-      name: Endpoint
-      description: The Baby Buddy endpoint to delete
-      required: true
-      selector:
-        select:
-          options:
-            - Children
-            - Changes
-            - Feedings
-            - Notes
-            - Sleep
-            - Temperature
-            - Timers
-            - Tummy Times
-            - Weight
-    entry:
-      name: Entry
-      description: If entry is Child, enter slug. Otherwise, enter ID.
-      required: true
-      selector:
-        text:
-
-services_changes_add:
-  name: Add Change
-  description: Adds a change
-  fields:
-    child:
-      name: Child
-      description: Child to add a change to
-      required: true
-      selector:
-        entity:
-    time:
-      name: Change time
-      description: Change time in YYYY-MM-DD HH:MM format
-      required: true
-      selector:
-        text:
-    wet:
-      name: Change wet
-      description: Was change wet
-      required: true
-      selector:
-        boolean:
-    solid:
-      name: Change solid
-      description: Was change solid
-      required: true
-      selector:
-        boolean:
-    color:
-      name: Change color
-      description: Change color
-      selector:
-        select:
-          options:
-            - Black
-            - Brown
-            - Green
-            - Yellow
-    amount:
-      name: Change amount
-      description: Change amount in number format
+    start:
+      name: Timer start time
+      description: Leave blank for a quick start
       selector:
-        text:
-    notes:
-      name: Change notes
-      description: Change notes
+        time:
+    name:
+      name: Timer name
+      description: Specify optional timer name
       selector:
         text:
 
-services_feedings_add:
+add_feeding:
   name: Add Feeding
-  description: Adds a feeding
+  description: Adds a feeding entry
+  target:
+    entity:
+      integration: babybuddy
+      domain: switch
```

Comment:
should this be switch?

Reply:
`add_feeding` service is part of the switch entity because a feeding can be linked to a timer (which is a switch in HA). If timer option is enabled in the service call the timer ID is used in the timer field in the api call. This allows to automatically assign the start/end times based on the active timer.